test_scripts/Resumption/InteriorVehicleData/025_InteriorVD_resumption_double_subscription_without_moduleId_unexpected_disconnect.lua

```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: SDL sends GetInteriorVehicleData response with WARNINGS result code
+--  in case of double subscription without moduleId after successful resuming on transport disconnect
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is subscribed to all modules via GetInteriorVehicleData(moduleType)
+-- 4. Unexpected disconnect and reconnect are performed
+-- 5. App reregisters with actual HashId after unexpected disconnect
+-- 6. HMI responds with SUCCESS resultCode to all requests from SDL
+--
+-- Sequence:
+-- 1. GetInteriorVehicleData(subscribe = true, module_1) is requested
+-- SDL does:
+-- - a. not send GetInteriorVehicleData request to HMI
+-- - b. send GetInteriorVehicleData ("WARNINGS") response to mobile App
```

Comment:
send GetInteriorVehicleData response (success=true, resultCode=WARNING, isSubscribed=true ) response to mobile App

Reply:
Done